src/main/scala/org/clulab/habitus/variables/CropAnalysis.scala

```diff
@@ -0,0 +1,63 @@
+package org.clulab.habitus.variables
+import org.clulab.utils.Closer.AutoCloser
+import org.clulab.odin.TextBoundMention
+import org.clulab.utils.{FileUtils, StringUtils, ThreadUtils}
+
+import java.io.{File, PrintWriter}
+
+object CropAnalysis {
+
+  def main(args: Array[String]): Unit = {
+    val props = StringUtils.argsToMap(args)
+    val inputDir = ""
+    val outputDir = ""
+    val threads = props.get("threads").map(_.toInt).getOrElse(1)
+
+    run(inputDir, outputDir, threads)
+  }
+
+  def run(inputDir: String, outputDir: String, threads: Int): Unit = {
+    new File(outputDir).mkdirs()
+
+    // fixme: temporary, simple text cleanup
+    def cleanText(text: String): String = text
+      .replace("\n", " ")
+      .replace("- ", "")
+
+    val processor = VariableProcessor()
+    val files = FileUtils.findFiles(inputDir, ".txt")
+    val parFiles = if (threads > 1) ThreadUtils.parallelize(files, threads) else files
+
+    new PrintWriter(new File(outputDir + "/mentions.tsv")).autoClose { pw =>
+      pw.println("filename\tmention type\tfound by\tsentence\tmention text\targs in all next columns (argType: argText)")
+      for (file <- parFiles) {
+        try {
+          val unfiltered = FileUtils.getTextFromFile(file)
+          val text = cleanText(unfiltered)
+          val filename = StringUtils.afterLast(file.getName, '/')
+          println(s"going to parse input file: $filename")
+          val parsingResults = processor.parse(text)
+          val targetMentions = parsingResults.allMentions
+          val contentMentions = targetMentions.filter(m => m.label == "CropAssignment" || m.label == "Crop")
```

Comment:
Is the move of `(!m.isInstanceOf[TextBoundMention]) {` to down below intentional?  It would change the printing slightly.  There is a TsvWriter class that one could use, but it isn't especially important.

Reply:
However, don't let my comment slow anyone down.  I'm fine to merge.

Reply:
Yes  please, it is intentional @kwalcock. Also I had this file here to work on the crop extraction analysis Mihai asked me to do.